lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -425,6 +439,32 @@ class ReportUIFeatures {
         this._toggleDarkTheme();
         break;
       }
+      case 'swap-locale': {
+        if (!this._swapLocaleOptions) throw new Error('must call .initSwapLocale first');
+
+        // TODO: esmodules would be nice here...
+        // @ts-ignore
+        if (!window.Lighthouse || !window.Lighthouse.i18n) await import('./i18n-module.js');
```

Comment:
the viewer is 216KB while i18n-module.js is 432KB.

Reply:
wowza without any locales too, is that just the i18n libs themselves?

Reply:
yea - altho I wasn't minimizing the latter. minimizes it's ~200